src/main/java/practice/StreamPractice.java

```diff
@@ -59,7 +87,14 @@ public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
      * return the names of all cats whose owners are women from `femaleAge` years old inclusively.
      */
     public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
-        return Collections.emptyList();
+
+        return peopleList.stream()
+                .filter(p -> p.getSex().equals(Person.Sex.WOMAN)
+                        && p.getAge() >= femaleAge)
+                .map(Person::getCats)
+                .flatMap(List::stream)
```

Comment:
```suggestion
                .flatMap(p -> p.getCatList().stream())
```
If you can use just 1 map, you should do it